app/models/publications/monthly_statistics/statistics_data_processor.rb

```diff
@@ -0,0 +1,32 @@
+module Publications
+  module MonthlyStatistics
+    class StatisticsDataProcessor
+      LOWER_STATS_THAT_VIOLATES_GDPR = 3
```

Comment:
Can we rename this? Maybe something like `MINIMUM_GDPR_COMPLIANT_TOTAL`

Reply:
Done